tools/asynq/cmd/stats.go

```diff
@@ -55,15 +59,21 @@ func init() {
 }
 
 type AggregateStats struct {
-	Active    int
-	Pending   int
-	Scheduled int
-	Retry     int
-	Archived  int
-	Completed int
-	Processed int
-	Failed    int
-	Timestamp time.Time
+	Active    int       `json:"active"`
+	Pending   int       `json:"pending"`
+	Scheduled int       `json:"scheduled"`
+	Retry     int       `json:"retry"`
+	Archived  int       `json:"archived"`
+	Completed int       `json:"completed"`
+	Processed int       `json:"processed"`
+	Failed    int       `json:"failed"`
+	Timestamp time.Time `json:"timestamp"`
+}
+
+type FullStats struct {
+	AggregateStats AggregateStats    `json:"aggregate"`
+	RdbStats       []*rdb.Stats      `json:"rdb"`
```

Comment:
(nit): Let's rename this field `QueueStats` and also json field name to `"queues"`